src/teams/teams.controller.ts

```diff
@@ -31,7 +31,7 @@ export class TeamsController {
     constructor(private readonly teamsService: TeamsService) {}
 
     @ApiOperation({
-        summary: "Gets all voyage teams.",
+        summary: "[Roles: Admin] Gets all voyage teams.",
```

Comment:
Do we have checklist for what is required when writing for swagger?

Reply:
Not atm the moment, but I can write one if you think it would be helpful